sdk/python/symbolchain/nem/MessageEncoder.py

```diff
@@ -0,0 +1,96 @@
+import secrets
+
+import cryptography
+
+from symbolchain.Cipher import AesCbcCipher, AesGcmCipher
+from symbolchain.CryptoTypes import PublicKey
+from symbolchain.nc import Message, MessageType
+from symbolchain.nem.KeyPair import KeyPair
+from symbolchain.nem.SharedKey import SharedKey
+
+
+class MessageEncoder:
+	"""Encrypts and encodes messages between two parties."""
+
+	def __init__(self, key_pair: KeyPair):
```

Comment:
i don't think we're using type hints anywhere else?

Reply:
should we _start_ using them? I'm getting much better completion with annotations